jaeger-core/src/test/java/io/jaegertracing/ConfigurationTest.java

```diff
@@ -137,7 +139,7 @@ public void testReporterConfigurationInvalidLogSpans() {
   public void testTracerTagslist() {
     System.setProperty(Configuration.JAEGER_SERVICE_NAME, "Test");
     System.setProperty(Configuration.JAEGER_TAGS, "testTag1=testValue1, testTag2 = testValue2");
-    io.jaegertracing.Tracer tracer = (io.jaegertracing.Tracer) Configuration.fromEnv().getTracer();
+    JaegerTracer tracer = (JaegerTracer) Configuration.fromEnv().getTracer();
```

Comment:
cast still needed?

Reply:
Yes, as `getTracer()` returns OT's Tracer and the test later calls `JaegerTracer#tags()`.